Gemfile.lock

```diff
@@ -0,0 +1,67 @@
+GIT
```

Comment:
Don't suppose there's any way the Danger and Gem files can be `.Something`'s so ppl can see they're not vital to runtime operations?

Reply:
Good point! I moved `Dangerfile`, `Gemfile` and `Gemfile.lock` to `.github`, since they are related to repo maintenance :smiley:

Reply:
oh, good, I'm glad it's flexible. `.scripts/` is another good place for them.

Reply:
I prefer under `.github` since they are not actual scritps. What do you think about the random LGTM picture? Should I keep it in?

Reply:
If you like, could rename `.scripts` to `.ci` since it just contains travis run/test scripts. But not vital.

I'd prefer dropping the random LGTM picture. Too big and obscures the actual result message.

Reply:
:+1: